cli/cmd/data/target.go

```diff
@@ -18,6 +18,7 @@ type TargetDetails struct {
 	Alpine        bool
 	DryRun        bool
 	Image         string
+  DockerPath    string
```

Comment:
please add spaces to match the other variables' indentation

Reply:
I added spaces to match indentation.